.github/workflows/dockerimage.yml

```diff
@@ -0,0 +1,46 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ Run2_2017 ]
+  pull_request:
+    branches: [ Run2_2017 ]
+
+env:
+  current_branch: $(echo ${{ github.ref }} | sed -E 's|refs/[a-zA-Z]+/||')
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[skip build]')"
+    steps:
+    - name: Look at key environment variables
+      run: echo -e "GITHUB_REF=${{ github.ref }}\nGITHUB_ACTOR=${{ github.actor }}\nGITHUB_REPOSITORY=${{ github.repository }}\nGITHUB_SHA=${{ github.sha }}\nBANCH=${{ env.current_branch }}"
+    - name: Take a look around
+      run: pwd && ls -alh
+    - name: Run a CVMFS Docker image and run the setup script
+      run: docker run -t -P --device /dev/fuse --cap-add SYS_ADMIN --security-opt apparmor:unconfined -e CVMFS_MOUNTS="cms.cern.ch" --name treemaker --entrypoint "/bin/bash" aperloff/cms-cvmfs-docker:light -c "/run.sh -c \"wget https://raw.githubusercontent.com/${{ github.repository }}/${{ github.sha }}/setup.sh && chmod +x setup.sh && ./setup.sh -a https -f ${{ github.actor }} -b ${{ env.current_branch }} -B -j 2\" && cvmfs_config wipecache"
+    - name: Commit the changes
+      run: docker commit -c 'ENTRYPOINT ["/run.sh"]' -c 'CMD []' treemaker treemaker/treemaker:${{ env.current_branch }}-latest
+    - name: Log into registry
+      run: echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
+    - name: Publish the new docker image
+      run: docker push treemaker/treemaker:${{ env.current_branch }}-latest
+  integration-test:
+    runs-on: ubuntu-latest
+    needs: [build]
+    if: "!contains(github.event.head_commit.message, '[skip test]')"
```

Comment:
can you update the readme so this is documented somewhere accessible?

Reply:
Done